virtual_ecosystem/models/animal/functional_group.py

```diff
@@ -82,6 +84,8 @@ def __init__(
         """The excretion type of the functional group."""
         self.migration_type = MigrationType(migration_type)
         """The migration type of the functional group."""
+        self.vertical_occupancy = VerticalOccupancy(vertical_occupancy)
```

Comment:
This will change simply to:
```suggestion
        self.vertical_occupancy = VerticalOccupancy.parse(vertical_occupancy)
```